CharacterCreation/03_Feats.txt

```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
 2d10 = 10, can be taken twice). If any of your die rolls is a 1 or 2, reroll
 the die.
 
+== Unarmed Combat Training ==
+
+Prerequisites:
+* STR 5, DEX 5
+* Levle 5, OR Assassin Class, OR Berserker Class
+
+	Increase base and minimum damage of Unarmed Strike by 20. 
+	
+	Additionally, Once per round after a successful melee strike, 
+if the target is not wielding a CQB weapon, you may use a reaction to 
+make a disarm attempt.
+
+== Unarmed Combat Expertise ==
+
+Prerequisites:
+* Unarmed Combat Training
+* STR 6, DEX 6
+* Level 10, OR Assassin Class and Level 5, OR Berserker Class and level 5
+
+	Increase base and minimum damage of Unarmed Strike by 20.

```

Comment:
I think we should either say 'additional 20 damage' or list the combined total of the damage from melee feats. It helps reduce that ambiguity.

Reply:
Can do